components/Navbar/index.js

```diff
@@ -0,0 +1,40 @@
+import Link from "next/link";
+import { useRouter } from "next/router";
+import { Nav, NavLink } from "../../styles";
+import { useEffect, useState } from "react";
+import VDWLogo from "../../public/VDW_Logo.png";
+import Image from "next/image";
+import { FiShoppingCart } from "react-icons/fi";
+
+const Navbar = () => {
+  const router = useRouter();
+  const [isSSR, setIsSSR] = useState(true);
+
+  useEffect(() => {
+    setIsSSR(false);
+  }, []);
+
+  return (
+    !isSSR && (
+      <header>
+        <Nav>
+          <Link href="/">
+            <NavLink active={router.pathname === "/" ? true : false}>
+              <p>Shop</p>
+            </NavLink>
+          </Link>
```

Comment:
`<Link>` already renders an `<a>` tag by default, so it shouldn't be necessary to use `<NavLink/>` inside `<Link>`